test/clj/rems/db/test_user_settings.clj

```diff
@@ -89,7 +89,7 @@
            (user-settings/validate-new-settings {:notification-email nil})))))
 
 (deftest test-notification-email-visible
-  (users/add-user! "pekka" {:eppn "pekka" :commonName "Pekka" :mail "pekka@example.com"})
+  (users/add-user! {:userid "pekka" :name "Pekka" :email "pekka@example.com"})
```

Comment:
Ia there on purpose :mail --> :email ?

Reply:
Yes, previously `add-user!` wanted raw attributes which use `:eppn` and `:mail`, now it takes formatted attributes which use `:userid` and `:email`. I guess we support `:mail` since that's what shibboleth gives us...